privaterelay/signals.py

```diff
@@ -1,15 +1,27 @@
+from datetime import datetime
+
 from django.core.exceptions import PermissionDenied
 
+from .models import Invitations
+
 
 ALLOWED_SIGNUP_DOMAINS = [
     'getpocket.com', 'mozilla.com', 'mozillafoundation.org'
 ]
 
 
-def mozillians_only(sender, **kwargs):
+def invitations_only(sender, **kwargs):
     sociallogin = kwargs['sociallogin']
     email = sociallogin.account.extra_data['email']
+    domain_part = email.split('@')[1]
     for allowed_domain in ALLOWED_SIGNUP_DOMAINS:
-        if email.endswith(allowed_domain):
+        if domain_part == allowed_domain:
             return True
-    raise PermissionDenied
+    try:
+        active_invitation = Invitations.objects.get(email=email, active=True)
```

Comment:
This `get` indicates that a user may access private relay even though the invitation was not sent: according to the `invitebetauser` command, an invitation's `active` field is set to `True` without `date_sent` being set. Is that intended?

Reply:
Yes. In addition to sending the big batch of email invitations to the Monitor wait-list, we are also adding some individual email addresses that we will invite "by hand" another way.